source/apiVolontaria/volunteer/models.py

```diff
@@ -44,6 +44,42 @@ def is_active(self):
     def __str__(self):
         return '{}'.format(self.name)
 
+    def generate_participation_report_data(self):
```

Comment:
I think you will need the same documentation here to explain what you do and return.

Reply:
Yes make sense, added.